include/roboteam_world/filters/BallObservation.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by rolf on 21-01-20.
+//
+
+#ifndef RTT_BALLOBSERVATION_H
+#define RTT_BALLOBSERVATION_H
+#include <roboteam_proto/messages_robocup_ssl_detection.pb.h>
+/**
+ * A struct to keep Ball Data and time as one observation.
+ */
+struct BallObservation {
```

Comment:
no namespace?

Reply:
Nothing in the repo is namespaced right now.

Reply:
Oh okay, approved then